neptune/new/sync.py

```diff
@@ -34,7 +34,7 @@
 from neptune.new.exceptions import (
     CannotSynchronizeOfflineExperimentsWithoutProject,
     NeptuneException,
-    ProjectNotFound,
+    ProjectNotFound, ExperimentNotFound,
```

Comment:
Import should be in newline

Reply:
You can check if such config will help:
![Screenshot from 2021-03-12 09-53-22](https://user-images.githubusercontent.com/3773836/110916276-ea66ee00-8318-11eb-8e41-088f1157ce4a.png)